File: ColorSudoku/src/main/java/sk/tuke/gamestudio/service/PlayerAccountService.java
package sk.tuke.gamestudio.service;

import sk.tuke.gamestudio.entity.PlayerAccount;

public interface PlayerAccountService {
    void addPlayerAccount(PlayerAccount playerAccount) throws PlayerAccountException;
    void resetPlayerAccount() throws PlayerAccountException;
    PlayerAccount getPlayerAccount(String game, String playerName) throws PlayerAccountException;
    PlayerAccount getLog(String game, String playerName, String password) throws PlayerAccountException;
}

File: frontend/src/_api/player_account.service.js
import {_exios} from "./index";

export const addPlayerAccount = (player, game, password) =>{
   return  _exios.post('/playerAccount', {player, game, password})
};

export const getPlayerAccount = (player, game) =>{
    return _exios.get('/playerAccount/getplayerAccount/' + game + '/' + player);
};

export const getLog = (game, player, password) =>{
    return _exios.get('/playerAccount/getlog/' + game + '/' + player + '/' + password);
};